testing-scripts/bigtable/test_bigtable.py

```diff
@@ -0,0 +1,104 @@
+"""
+This module provides testing functionality of the BigTable Init Action.
+
+Test logic:
+1. Create test table and fill it with some data by injecting commands into hbase shell.
+2. Validate from local station that BigTable has test table created with right number of rows.
+
+Note:
+    Test REQUIRES cbt tool installed which provides CLI access to BigTable instances.
+    See: https://cloud.google.com/bigtable/docs/cbt-overview
+"""
+import unittest
+import random
+
+from parameterized import parameterized
+
+from dataproc_test_case import DataprocTestCase
+
+
+class BigTableTestCase(DataprocTestCase):
+    COMPONENT = 'bigtable'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/bigtable/bigtable.sh'
+    TEST_SCRIPT_FILE_NAME = "run_hbase_commands.py"
+    METADATA = None
+    DB_NAME = None
+    ZONE = None
+
+    def setUp(self):
+        super().setUp()
+        self.DB_NAME = "test-{}-db".format(random.randint(1, 10000))
```

Comment:
If you `Ctrl-C` a test, you will end up leaking Bigtable instances. It would be nice to use a consistent name, and call tearDown in setUp

Reply:
Wouldn't that break multiple people running the tests concurrently? I'll see if I can add a TTL to the DB.

Reply:
Ack, good point